tools/__init__.py

```diff
@@ -14,29 +14,30 @@
 #  You should have received a copy of the GNU Lesser General Public
 #  License along with this library.
 
-import logging
+from __future__ import print_function
```

Comment:
Unused ?

Reply:
used to locally redefine print (to print in sys.err)